hedera-node/hedera-mono-service/src/main/resources/bootstrap.properties

```diff
@@ -222,7 +222,7 @@ hedera.prefetch.codeCacheTtlSecs=600
 utilPrng.isEnabled=true
 tokens.autoCreations.isEnabled=true
 accounts.blocklist.enabled=false
-accounts.blocklist.resource=
+accounts.blocklist.path=
```

Comment:
Do we not have default path for this ?

Reply:
This is the second case from the PR description:

> If the path does not correspond to an existing file or it's left empty then we'll fallback and load the default src/main/resources/evm-addresses-blocklist.csv.

So leaving this empty means we will default to loading the `src/main/resources/evm-addresses-blocklist.csv`.